test/sharness/t0250-files-api.sh

```diff
@@ -260,6 +260,12 @@ test_files_api() {
     verify_dir_contents /cats/this/is/a/dir
   '
 
+  test_expect_success "dir has correct name" '
+    echo "this/" > ls_dir_expected &&
+    ipfs files ls -l /cats | grep this/ | awk '\''{print $1}'\'' > ls_dir_actual &&
```

Comment:
I think the standard way is to output "filename hash size" in the `ls_dir_actual` instead of using `awk`+`grep`.

Reply:
@schomatis yeah, I also think that "filename hash size" is a better way. However, the `hash` is unfixed, because `ipfs files mkdir` uses different `arg`: 
https://github.com/ipfs/go-ipfs/blob/9bf4e4145ea897ced2f9aa2087296b3beda6e941/test/sharness/t0250-files-api.sh#L109-L116

Reply:
We should then save the dir hash generated on every call, although I'm not sure if that's the best approach, @magik6k WDYT?

Reply:
yep, we should grab the generated hash, save it and compare that